### DISCUSSION TITLE
Adding Fleet server to the mix

### DISCUSSION THREAD

**Discussion by benisai** [sherifabdlnaby/elastdocker#52] [Questions & Troubleshooting]
I'm trying to get Fleet server running in a docker along side the other elastic containers this project, but I'm having a hard time. 
Is there any way you can add fleet server to the mix with your expert knowledge? 
I tried to follow this [guide](https://www.gooksu.com/2021/09/fleet-server-with-elasticsearch-in-docker-container/), but it just didnt work.

**Comment by simwai** [comment]
I swear I would love that feature too

**Comment by btmndkh** [comment]
me three
